sapnwbootstrap-formula.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Mar  3 09:52:45 UTC 2020 - Xabier Arbulu <xarbulu@suse.com>
+
+- Version 0.2.5
```

Comment:
@arbulu89 Don't forget to include the bug number here ;)

Reply:
I think is better to just add the bug trackers when we are going to ship to SLE. I don't think we will ship all the updated versions... So when we have enough worthy changes to be released, we can update them the changes file. What do you think?

About adding more resources, by now we only support the IP management with routes (not alias). This is because the other usage would require different infrastructure creation, and by now, with terraform we only create the routes way. We could implement and give the 2 options in the future, that's true